docs/source/tutorials_source/platform/tutorial_label_studio_export.py

```diff
@@ -35,32 +35,35 @@
     :align: center
     :alt: Import dialog of LabelStudio.
 
-Export a tag in the LabelStudio format
-------------------
-Now go again to the tab with the Lightly webapp.
-Just head over to the *Download* tag to see the different download options.
-Choose 'Export Reduced Dataset', an expiration duration giving you
-enough time to label all image and choose the form 'json'.
+Export from Lightly in the LabelStudio format
+---------------------------------------------
+Now go again to the tab with the `Lightly Platform <https://app.lightly.ai>`_.
```

Comment:
There is no mention of a "tab" beforehand so I would follow the suggestion I made about "Open up your dataset in the lightly platform..."